src/js/subtraction/types.ts

```diff
@@ -0,0 +1,66 @@
+import { JobMinimal } from "../jobs/types";
+import { SampleNested } from "../samples/types";
+import { UserNested } from "../users/types";
+import { SearchResult } from "../utils/types";
+
+/* The composition of a Nucleotide */
+export type NucleotideComposition = {
+    a: number;
+    c: number;
+    g: number;
+    t: number;
+    n: number;
+};
+
+/* A Subtraction file */
+export type SubtractionFile = {
+    download_url: string;
+    id: number;
+    name: string;
+    size: number;
+    subtraction: string;
```

Comment:
```suggestion
    subtraction: string;
```
I think this is probably a subtraction id but probably worth double checking and adding a docstring to the effect